cypress/integration/tests/administrator/proxy/proxy.enable.disable.test.ts

```diff
@@ -32,42 +33,49 @@ describe("Proxy operations", () => {
 
     it("Http Proxy port and host field validation", function () {
         Proxy.open();
-        proxy.enableSwitch(ProxyViewSelectors.httpSwitch);
-        proxy.fillHost(ProxyType.http, proxy.hostname);
-        proxy.fillPort(ProxyType.http, "Invalid port");
+        httpProxy.enable();
+        httpProxy.fillHost(getRandomWord(121));
+        httpProxy.fillPort("Invalid port");
         cy.get(ProxyViewSelectors.portHelper).contains("This field is required");
+        cy.get(ProxyViewSelectorsByType[httpProxy.type].hostHelper).contains(
+            "This field must contain fewer than 120 characters."
+        );
         cy.get(submitButton).should("be.disabled");
-        proxy.disableSwitch(ProxyViewSelectors.httpSwitch);
+        httpProxy.unConfigureProxy();
     });
 
     it("Https Proxy port and host field validation", function () {
         Proxy.open();
```

Comment:
We can remove this line now.

Reply:
Solved in f3e121befe9d79ea59ad8bee2368760ff8b64270.